carapace-server/src/test/java/org/carapaceproxy/SimpleHTTPProxyTest.java

```diff
@@ -161,6 +164,13 @@ public void testSsl() throws Exception {
 
             stats = server.getConnectionsManager().getStats();
             assertNotNull(stats.getEndpoints().get(key));
+
+            // check tls proto available versions (expected TSLv1.2, TSLv1.3)
```

Comment:
I can't understand this test.

I feel we should have two kind of tests:
- for each protocol in the default list start a listener and connect to it using a client, verifying the actual tls protocol
- verify that if you configure an unknown protocol the listener fails to start